Source/WebCore/dom/ios/PointerEventIOS.cpp

```diff
@@ -72,13 +72,10 @@ PointerEvent::PointerEvent(const AtomString& type, const PlatformTouchEvent& eve
     , m_pointerType(event.touchTypeAtIndex(index) == PlatformTouchPoint::TouchType::Stylus ? penPointerEventType() : touchPointerEventType())
     , m_isPrimary(isPrimary)
 {
-    // See https://github.com/w3c/pointerevents/issues/274. We might expose the azimuth and altitude
-    // directly as well as the tilt.
-    double azimuthAngle = event.azimuthAngleAtIndex(index);
-    double altitudeAngle = event.altitudeAngleAtIndex(index);
-
-    m_tiltX = round(cos(azimuthAngle) * cos(altitudeAngle) * 90);
-    m_tiltY = round(sin(azimuthAngle) * cos(altitudeAngle) * 90);
+    m_azimuthAngle = event.azimuthAngleAtIndex(index);
+    m_altitudeAngle = m_pointerType == penPointerEventType() ? event.altitudeAngleAtIndex(index) : piOverTwoDouble;
```

Comment:
The introduction of a `m_pointerType != penPointerEventType()` case is a behavior change for `m_tiltX` and `m_tiltY`. Is that expected?

cc @aprotyas

Reply:
I believe so, at the moment tiltY returns 90 by default when it should return 0. This changes makes it so that both tiltX and tiltY are 0 by default

Reply:
Hmm, I'm not following along, why do we specialize based on the pointer type here?